src/dlsproto/client/DlsClient.d

```diff
@@ -744,6 +744,38 @@ public class DlsClient : IClient
         this.node_handshake = new NodeHandshake;
     }
 
+     /***************************************************************************
+
+        Constructor with support for only the neo protocols. Accepts only
+        `Neo.Config`, not `IClient.Config`, for applications that use the neo
+        protocol only.
+
+        Params:
+            epoll = EpollSelectDispatcher instance to use
+            neo_config = swarm.neo.client.mixins.ClientCore.Config instance.
+                (The Config class is designed to be read from an application's
+                config.ini file via ocean.util.config.ConfigFiller.)
+            conn_notifier = delegate which is called when a connection attempt
+                succeeds or fails (including when a connection is
+                re-established). Of type:
+                void delegate ( IPAddress node_address, Exception e )
+            conn_limit = maximum number of connections to each DLS node
+            queue_size = maximum size of the per-node request queue
+            fiber_stack_size = size of connection fibers' stack (in bytes)
+
+    ***************************************************************************/
+
+    public this ( EpollSelectDispatcher epoll, Neo.Config neo_config,
+        Neo.ConnectionNotifier conn_notifier,
+        size_t conn_limit = IClient.Config.default_connection_limit,
```

Comment:
I'm probably forgetting some reasoning: why allow these legacy-only things to be configured?

Reply:
I don't think there's need for them to be configured, really.